api/server.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/get', (req, res) => {
+  res.status(200).json(data);
+});
```

Comment:
Nice and easy GET endpoint added to verify data is being successfully posted and deleted.